lib/ocpp/v16/charge_point_impl.cpp

```diff
@@ -993,7 +993,14 @@ void ChargePointImpl::message_callback(const std::string& message) {
                 if (enhanced_message.messageType == MessageType::BootNotificationResponse) {
                     this->handleBootNotificationResponse(json_message);
                 } else {
-                    this->handle_message(enhanced_message);
+                    if (enhanced_message.messageType == MessageType::GetConfiguration ||
+                        enhanced_message.messageType == MessageType::ChangeConfiguration ||
+                        enhanced_message.messageType == MessageType::TriggerMessage) {
```

Comment:
I think this is too restrictive, the 1.6 spec in section 4.2 states

> While in pending state, the following Central System initiated messages are not allowed:
RemoteStartTransaction.req and RemoteStopTransaction.req

There are no other restriction (other than not issuing chargepoint initiated requests)

Reply:
@bWF0dGhpYXMK what's your opinion on this?

Reply:
Hi, a review of the latest code, there is a PR https://github.com/EVerest/libocpp/pull/520 managing that case with an answer on Rejected. From the ocpp1.6, for me it is not clear what means not allowed, initially I thought on "no" answer, which is two edged. Because an on the RPC framework point of view you require at least an answer such as Call Error. 
So on that level, i think this PR can be closed. Either reject or CALL Error.

It remains the clarification on ReserverNow.req, Reset.req, ... what behavior here is the most likely in the sense of the describtion. 

From this " The Central System MAY send request messages to retrieve information from the
Charge Point or change its configuration. The Charge Point SHOULD respond to these messages.". It sound like behave as in Remote(Start/Stop).req...